messages/faq_visiting.md

```diff
@@ -1,3 +1,3 @@
 ## Visiting or transferring to VATSIM Scandinavia
 - To transfer, please refer to [this guide](https://wiki.vatsim-scandinavia.org/books/getting-started-AVr/chapter/joining-vatsim-scandinavia).
-- To become a visiting controller, [check out this guide](https://wiki.vatsim-scandinavia.org/books/training-documents/page/transfer-and-visiting-policy-in-vatsim-scandinavia).
\ No newline at end of file
+- To become a visiting controller, check out [this guide](https://wiki.vatsim-scandinavia.org/books/training-documents/page/transfer-and-visiting-policy-in-vatsim-scandinavia).
```

Comment:
```suggestion
- To become a visiting controller, [check out this guide](https://wiki.vatsim-scandinavia.org/books/training-documents/page/transfer-and-visiting-policy-in-vatsim-scandinavia).
```

disagree: I think best practice is to link the verb as well rather than just the target (e.g. why "click _here_" isn't a useful link for context or semantics, accessability, etc.)

Reply:
Thanks for following up! I prefer the latter to the former. WDYT, @Blt950?

Reply:
If we pull the accessibility card here, I'd say neither options describe where the link goes. What about just rewriting the whole thing and making the two options even more visible?

> ## Visiting or transferring to VATSIM Scandinavia
> 
> ### Transfer your account
> If you wish to move your account to VATSIM Scandinavia, [follow this step-by-step transfer instruction.](https://wiki.vatsim-scandinavia.org/books/getting-started)
> 
> ### Become a visiting controller
> If you want to control as a visitor, [read our visiting and transfer policy.](https://wiki.vatsim-scandinavia.org/books/training-documents/page/transfer-and-visiting-policy-in-vatsim-scandinavia)